Clinical-Genomics-Lund/gens
# Set default annotation track

Set the default annotation track by passing the annotation argument when opening a result in Gens. If no default is set the first annotation file in the drop down menu is selected.

This PR also refactors Gens to a python package and adds a change log document.

**How to test this**
Verify the expected functionality for the use cases, a default is set, no default is set and a non-existing annotation is set.
1. Example of setting `CAGdb.all.200303-hg38.aed` as the default annotation; http://10.0.224.64:5003/1883-20-5-1?region=8:197901-2247000&hg_type=38&annotation=CAGdb.all.200303-hg38.aed
2. Example of not setting a annotation,  http://10.0.224.64:5003/1883-20-5-1?region=8:197901-2247000&hg_type=38
3. Example of selecting a non-existing track, http://10.0.224.64:5003/1883-20-5-1?region=8:197901-2247000&hg_type=38&annotation=foobar
